andela/phoenix-bn-backend
# #167765404 User Role Settings (CRUD)

###What does this PR do?
Have the super administrator set a role for a user

### Description of task to be completed
`POST /api/v1/auth/signup`
`POST /api/v1/role`
- Write controller method in (src/controllers/userController.js) that would use the Roles models to set user's role after registration
- write controller method in `scr/controllers/RolesController.js` that would set a role for an existing user
- Write unit tests
- Document with swagger
- review app link https://phoenix-bn-backend-stagi-pr-41.herokuapp.com/api/v1

### How should this be manually tested?
Checkout to the branch and run `npm run devstart` to check for feature
Checkout to the branch and run `npm test` for tests

### What are the relevant pivotal tracker stories?
#167765404

### Implementation Plan
https://docs.google.com/document/d/1kzTaMyYSOWr0R-XRkYK_w7vnb54nfo8q1i1zB088jw8/edit?usp=sharing
